index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" 
+    integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" /> 
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" 
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"> 
+    <link rel="stylesheet" href="style.css">
+    <title>PyLadies Namibia Workshop 2023</title>
+</head>
+
+<body>
+    <!-- Toolbar Section -->
+    <header class="header">
+        <div class="header-top-bar">
+            <ul class="header-top-content">
+                <li><i class="fab fa-facebook-f"></i></li>
+                <li><i class="fab fa-twitter"></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <div class="navigation">
+            <div class="logo">
+                <a href="index.html">
+                    <img src="images/logo.jpg" alt="logo">
+                </a> 
+            </div>
+            <div id="union" class="burger fas fa-bars"></div>
+            <nav class="mobile-menu">
+                <a class="menu-link" href="about.html">About</a>
+                <a class="menu-link" href="#">Program</a>
+                <a class="menu-link" href="#">Join</a>
+                <a class="menu-link" href="#">Sponsor</a>
+                <a class="menu-link" href="#">News</a>
+                <a class="last-menu-link" href="#">PyLadies Campaign</a>
+            </nav>
```

Comment:
It'd be better to make the menu take the full width of the screen on the mobile screen, since it's getting cut in half 🤗

![image](https://user-images.githubusercontent.com/73714615/190581382-a8b4b440-1d0f-44dd-b876-58089690ee24.png)